src/src/attendances/attendances.core.service.ts

```diff
@@ -241,10 +241,17 @@ export class AttendancesCoreService {
 				  }
 			  }`;
 		try {
-			const data_list = (
-				await this.hasuraServiceFromServices.getData({ query })
-			)?.data?.attendance;
-			return data_list;
+			const result_response =
+				await this.hasuraServiceFromServices.getData({ query });
+			console.log('result_response', result_response);
+			const data_list = result_response?.data?.attendance;
+			//console.log('data_list cunt------>>>>>', data_list.length);
```

Comment:
remove console logs pls

Reply:
@manojLondhe  console log on dev is required for testing and track cron job flow. after testing complete on dev server then i will remove all logs and sent PR again

Reply:
@manojLondhe  our previous face rekognition 3 Cron Job contains console.log to debug issues.